custom/opm/reports.py

```diff
@@ -429,18 +416,19 @@ def _service_dates(self):
             }
         }
         """
```

Comment:
Do a couch lookup to get all VHND forms, then assemble that same data structure based on the forms.